config/routes.rb

```diff
@@ -1,10 +1,6 @@
 Rails.application.routes.draw do
```

Comment:
- [ ] Great on your project so far 👏. However, the routes are not handled based on the wireframes. There, the routes are as follows: :point_down: which means a post is owned by the user and that the posts routes are embedded with the users routes. 

![Screenshot from 2023-11-13 18-37-55](https://github.com/Xghayor/threads-rails/assets/89228743/80a4ef34-8cd9-4030-847d-cd0475b4ba07)

At this moment, your posts routes are separate from the users'. Please ensure to fix that by updating your routes as follow: :point_down: 

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```